CMakePresets.json

```diff
@@ -159,18 +159,26 @@
         "release"
       ]
     },
+    {
+      "name": "clang-base",
+      "environment": {
+        "COMPILER_CXX_FLAGS": "$env{COMPILER_C_FLAGS} -stdlib=libc++"
+      },
+      "inherits": "unix-sanitizer",
+      "hidden": true
+    },
```

Comment:
this is not an issue in our repo but an incompatibility in libstdc++

see also https://www.mail-archive.com/debian-bugs-dist@lists.debian.org/msg1918839.html and https://stackoverflow.com/questions/76859275/error-compiling-a-cpp-containing-stdchrono-errorstatic-constexpr-unsigned-fra